apis/infix-en_GB/atrium-api-infix-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/ListAssertionSamples.kt

```diff
@@ -0,0 +1,73 @@
+package ch.tutteli.atrium.api.infix.en_GB.samples
+
+import ch.tutteli.atrium.api.infix.en_GB.*
+import ch.tutteli.atrium.api.verbs.internal.expect
+import ch.tutteli.atrium.translations.DescriptionComparableAssertion
+import kotlin.test.Test
+
+class ListAssertionSamples {
+    private val isLessThanDescr = DescriptionComparableAssertion.IS_LESS_THAN.getDefault()
+    private val isGreaterThanDescr = DescriptionComparableAssertion.IS_GREATER_THAN.getDefault()
+
+    @Test
+    fun getFeature() {
+        val list = listOf(1, 2, 3)
+
+        expect(list) get 0 isLessThan 2
+        //            |        | subject is still of type Int (still 1)
+        //            | subject is now of type Int (actually 1)
+
+        expect(list) get 0 isGreaterThan 0
+        //            |         | subject is still of type Int (still 1)
+        //            | subject is now of type Int (actually 1)
+
+        fails {
+            expect(list) get 3 isLessThan 0
+        //                |        | not reported
+        //                | fails because index 3 is out of bound
+        // use `get index(elementIndex) { ... }` if you want that all assertions are evaluated
+        } message {
+            contains("index out of bounds")
+            containsNot("is less than: 2")
+        }
+
+        fails {
+            expect(list) get 0 isGreaterThan 2     //fails
+            expect(list) get 0 isLessThan 0        // not reported because `isGreaterThan 2` already fails
+        // use `get index(elementIndex) { ... }` if you want that all assertions are evaluated
+
+        } message {
+            contains("${isGreaterThanDescr}: 2")
+            containsNot("${isLessThanDescr}: 0")
+        }
+    }
+
+    @Test
+    fun get() {
+        val list = listOf(1, 2, 3)
+
+        expect(list) get index(0) {           // subject inside this block is of type Int (actually 1)
+            isLessThan(2)
```

Comment:
That's not the infix API (same same for other places). Should be:
```suggestion
            it isLessThan 2
```